src/lumen/hardware/single_led_button.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "lumen/hardware/led_button.hpp"
```

Comment:
Why is this including itself?

Reply:
This isn't including itself. It's including the base class.